msgpack5/index.d.ts

```diff
@@ -0,0 +1,51 @@
+// Type definitions for msgpack5 v3.4.0
+// Project: https://github.com/mcollina/msgpack5/
+// Definitions by: Wonshik Kim <https://github.com/wokim/>, Kei Son <https://github.com/heycalmdown/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import stream = require('stream');
+import bl = require('bl');
+  
+interface Options {
+  forceFloat64: boolean;
+  compatibilityMode: boolean;
+}
+
+declare namespace msgpack5 {
+  interface Options {
+    header?: boolean;
+    msgpack?: any;
+  }
+
+  class Base extends stream.Transform {
+    protected _header: any;
+    protected _msgpack: any;
+  }  
+
+  class Encoder extends Base {
+    constructor(opts?: Options);
+    public _transform(obj: any, enc: any, done: Function): any;
+  }
+
+  class Decoder extends Base {
+    constructor(opts?: Options);
+    public _chunks: bl;
+    public _length: number;
+    public _transform(buf: any, enc: any, done: Function): any;
+  }
+
+  interface MessagePack {
+    encode(obj: any): bl;
+    decode<T>(buf: Buffer): T;
```

Comment:
See the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes) about `getMeAT<T>(): T`.